internal/ports/same_peer_switch_test.go

```
@@ -33,7 +34,7 @@ func TestSamePeerSwitchWireStrict(t *testing.T) {
 			decode:  func(payload []byte) (any, error) { return UnmarshalSamePeerSwitchRequest(payload) },
 			assert: func(t *testing.T, decoded any) {
 				t.Helper()
-				if got := decoded.(SamePeerSwitchRequest); got != request {
+				if got := decoded.(protocol.SamePeerSwitchRequest); got != request {
```

golangci-lint (2.12.2)

[error] type assertion must be checked

(forcetypeassert)